PyPDF2/_reader.py

```diff
@@ -1058,16 +1051,11 @@ def get_object(self, indirect_reference: IndirectObject) -> Optional[PdfObject]:
             # override encryption is used for the /Encrypt dictionary
             if not self._override_encryption and self.is_encrypted:
                 # if we don't have the encryption key:
-                if not hasattr(self, "_decryption_key"):
+                if not hasattr(self, "_encryption"):
```

Comment:
Why did you change the name?

Reply:
it's not just the key, it's the object representing all the encrypt and decrypt staff.